deployment/v3/utils/prop_migrator/Overview.md

```diff
@@ -0,0 +1,225 @@
+# README - Structure and Design
```

Comment:
Guess we can let the document heading be README

and then we can ## Structure and Design in the next line

Please add a sentence informing what the document covers (large picture) and then start with explaining the scripts.